### PR TITLE
Merged all the high contrast themes files into one theme file

### PR DESCRIPTION
**Pull request by himgoyalmicro**
## Description

1. Bring parity with the WinUI Styles. 
2. Reducing redundancy by combining all different High Contrasts files into one. 
3. Allowing customization of High Contrast Themes 

## Regression

NA

## Testing

Build passed and tested on sample application

## Risk

Low Risk 

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/dotnet/wpf/pull/9336)

### CODE REVIEW

File: src/Microsoft.DotNet.Wpf/src/Themes/PresentationFramework.Fluent/Resources/Theme/HC.xaml
Comment (hiteshwpfmsft) [inline src/Microsoft.DotNet.Wpf/src/Themes/PresentationFramework.Fluent/Resources/Theme/HC.xaml:1]: Have we autogenerated this file?

Comment (lindexi) [reply src/Microsoft.DotNet.Wpf/src/Themes/PresentationFramework.Fluent/Resources/Theme/HC.xaml:1]: @hiteshwpfmsft How about [src/Microsoft.DotNet.Wpf/src/Themes/Generator/ThemeGenerator.Fluent.ps1](https://github.com/dotnet/wpf/pull/9293/files#diff-ea4932dd6da36ffaf4619e0fdfc363c246d456a3b2175da3de1459eda256971dR1)

Comment (dipeshmsft) [reply src/Microsoft.DotNet.Wpf/src/Themes/PresentationFramework.Fluent/Resources/Theme/HC.xaml:1]: This has not been auto generated, while merging the changes from wpfui, we brought the files here. Although, I will not extend the script to create HC theme file for fluent.

**Pull request closed without merging**